integration/restart_test.go

```diff
@@ -195,150 +195,3 @@ func TestContainerdRestart(t *testing.T) {
 		assert.True(t, found, "should find image %+v", i1)
 	}
 }
-
-// Note: The test moves runc binary.
-// The test requires:
-// 1) The runtime is runc;
-// 2) runc is in PATH;
-func TestUnknownStateAfterContainerdRestart(t *testing.T) {
```

Comment:
maybe a todo or issue to develop an integration test for unknownstate

Reply:
Done

Reply:
https://github.com/containerd/cri/issues/1310